lib/ansible/galaxy/collection.py

```diff
@@ -356,6 +378,45 @@ def from_name(collection, apis, requirement, force, parent=None):
         return req
 
 
+def get_available_api_versions(galaxy_api):
+    headers = {}
+    headers.update(galaxy_api._auth_header(required=False))
+
+    url = _urljoin(galaxy_api.api_server, "api")
```

Comment:
You need to fix up your API endpoints to actually return `v2`, right now all of these endpoints (dev, qa, prod) only return `v1` https://galaxy.ansible.com/api/.